RottingOranges.java

```diff
@@ -63,7 +62,7 @@ private void dfs(int[][] grid, int m, int n, int i, int j, int[][] dirs, int tim
 }
 // -----------------------BFS------------------------------------
 // Time Complexity :O(mn)
-// Space Complexity :min of m and n
+// Space Complexity :O(mn) space will be min of m and n in case rotten orange is at corner and all other are fresh
```

Comment:
Here the space complexity will be O(mn) in the worst case. For example lets consider all the oranges are rotten except one orange. Then all the (m*n) - 1 oranges will be stored in side the queue.